environment.yml

```diff
@@ -15,6 +15,7 @@ dependencies:
     - contextily
     - geopandas
     - ipython
+    - panel
```

Comment:
I agree with @weiji14's comment https://github.com/GenericMappingTools/pygmt/pull/2498/files#r1168046439, so `panel` should be a "dev dependency" related to documentation building. Thus it should be moved to the bottom of the list.

Reply:
This is updated in commit https://github.com/GenericMappingTools/pygmt/pull/2498/commits/66b0087a806c8e0b7bff005af0004d1bd74c061f.